core/kazoo_services/src/kazoo_services_maintenance.erl

```diff
@@ -149,7 +149,11 @@ js_quantify_map() ->
        "        case 'user':"
        "            emit(['users', doc.priv_level], 1);"
        "            if (doc.qubicle && doc.qubicle.enabled) {"
-       "                emit(['qubicle', 'recipients'], 1);"
+       "                if (doc.qubicle.pvt_prod_level == 'pro') {"
```

Comment:
@mark2600 we would not want to embed a pvt_ parameter in an object.  These need to be at the root level for the the functions like kz_doc:public_fields to work.  I recommend doc.pvt_qubicle_offering.

Reply:
@mark2600 actually, I just realized that this is emitted when processing the user document so you wouldn't know the pvt_qubicle_offering on the queue document.  However, I assume that you must be storing the parameters differently on the user document... Something like:

` 
  "qubicle": {
    "presence_id": "karl-support",
    "default_device": "75cc3551a8e3b0702b2131040c430628",
    "recipient": {
      "basic": {
        "enabled": true
      },
      "pro": {
        "enabled": true
      }
    }
  }
`

That would also make it easy to iterate the keys of the 'recipient' object and emit any enabled...

if (doc.qubicle && doc.qubicle.recipient) for (offering in doc.qubicle.recipient) if (doc.qubicle.recipient[offering].enabled) emit(['qubicle', offering + '_recipient'], 1);;